tests/scripts/task_unit_test.sh

```diff
@@ -16,21 +16,27 @@ if [ $# -ne 2 ]; then
     fail "Error: must specify backend and device"
 fi
 
+TEST_DEVICE=$2
```

Comment:
There is no need to modify the old test scripts. Just add a new script and test cugraph related UT.

Reply:
I added a new directory to test cugraph related UT only, though I utilize existing `task_unit_test.sh`. An extra `test.sh` is very like to existing one. What do you think?

Reply:
Having a lot of if/else in `task_unit_test.sh` makes it hard to maintain in the future. Please add a new script.

Reply:
done